src/app/models/AbstractModel.spec.ts

```diff
@@ -132,6 +132,67 @@ describe("AbstractModel", () => {
     });
   });
 
+  describe("nullEmit", () => {
+    function hasFormDataOnlyAttributes(model: AbstractModel) {
+      return model.hasFormDataOnlyAttributes({ create: true });
+    }
+    const testFile = new File([""], "testFileName.png");
+
+    it("should emit null type in JSON attributes", () => {
+      class Model extends MockModel {
+        @bawPersistAttr({ create: true, supportedFormats: ["json", "formData"] })
+        public image: any;
+      }
+      const model = new Model({ id: 1, image: null });
+      expect(Object.keys(model.getJsonAttributes({ create: true }))).toContain("image");
```

Comment:
This test could emit any value other than `null` and it wouldn't fail.

```suggestion
      var actual = model.getJsonAttributes({ create: true });
      expect(actual).toHaveOwnProperty("image");  // not sure if this API exists
      expect(actual.image).toBe(null);
      // or 
      expect(actual).toEqual(jasmine.objectContaining({
        image: null
      }));
```

Reply:
hasOwnProperty